proposals/lip-0046.md

```diff
@@ -196,7 +196,10 @@ This function returns `True` if `seedReveal` was correctly revealed, otherwise,
 It is specified as:
 
 ```python
-def isSeedRevealValid(generatorAddress: Address, seedReveal: bytes) -> bool:
+def isSeedRevealValid(generatorAddress: Address, blockAssets: BlockAssets) -> bool:
+    # let `blockAssetBytes` be the bytes included in the `blockAssets` for the Random module           
```

Comment:
Perhaps we should add some explanation on how do we find this particular entry of the `blockAssets` array (check implementation?).

Reply:
We need to add the asset.
https://github.com/LiskHQ/lisk-sdk/blob/5613c8fd6df83132eb7528a90c20fe837375ef0e/framework/src/state_machine/state_machine.ts#L82

I do not think we need an explanation here.

Reply:
I am not sure that I understand.

Reply:
I agree with Vikas on that we do not necessarily need to provide all the technical details here. I hope it should be clear enough for the devs.